### DISCUSSION TITLE
how to uninstall openwebui or get the administrator ID (not PASSWORD)?

### DISCUSSION THREAD

**Discussion by bookbindinggithub** [open-webui/open-webui#3812] [Q&A]
When I update. 
The account ID and password recorded in the browser are no longer available. 
for old vision,  didn't use @ to register an account, so I couldn't log in with administrator to allow a new user account, 
till now.  i can't use open webui anymore. 
there are many way to change password, but i don't know the account ID.
who can save me to uninstall openwebui or get the administrator ID (not PASSWORD)?
I have tryed remove open webui on Docek, but , restart the computer, It reappeared. I don't have a way to discern anymore.
# who can help me to save my open webui?

**Comment by pleabargain** [comment]
I'm having the same problem.
